client/src/Components/Custom Title/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Typography, Image } from 'antd';
+import PropTypes from 'prop-types';
+import linesRight from '../../assets/images/linesRight.svg';
+import linesLeft from '../../assets/images/linesLeft.svg';
+
```

Comment:
Import the style file here
```suggestion
import './style.css';

```